python-numpy-basics/operators_and_conrol_statements.ipynb

```diff
@@ -0,0 +1,635 @@
+{
```

Comment:
<p>@ninja3011, can u check example no. 4 and 8 here</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/SRA-VJTI/Intro_to_CV/pull/9/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='6'/>